api/src/main/java/io/kafbat/ui/controller/KafkaConnectController.java

```diff
@@ -285,4 +285,26 @@ private Comparator<FullConnectorInfoDTO> getConnectorsComparator(ConnectorColumn
       default -> defaultComparator;
     };
   }
+
+  @Override
+  public Mono<ResponseEntity<Void>> resetConnectorOffsets(String clusterName, String connectName,
+      String connectorName,
+      ServerWebExchange exchange) {
+    ConnectAction[] connectActions;
+
+    connectActions = new ConnectAction[] { ConnectAction.VIEW, ConnectAction.RESET_OFFSETS };
+
+    var context = AccessContext.builder()
+        .cluster(clusterName)
+        .connectActions(connectName, connectActions)
```

Comment:
more of a code style preference, imo it'd look better as a list of static imports:
`.connectActions(connectName, VIEW, RESET_OFFSETS)`, what do you think?

```suggestion
        .connectActions(connectName, VIEW, RESET_OFFSETS)
```

Reply:
I agree, that's much better. I'll do the change.